FakeItEasy.sln

```diff
@@ -74,6 +74,14 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{2EED
 		samples\FakeItEasy.Examples.ruleset = samples\FakeItEasy.Examples.ruleset
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "analyzer", "analyzer", "{86990EDA-25B4-4BFC-BC44-14D1B754261A}"

```

Comment:
Do we want this folder, or should the analyzer project live under `src`? The analyzer test project lives under `tests`.